sheet.go

```diff
@@ -78,10 +78,17 @@ func (s *Sheet) Col(idx int) *Col {
 // containing the data from the field "A1" on the spreadsheet.
 func (sh *Sheet) Cell(row, col int) *Cell {
 
-	if len(sh.Rows) > row && sh.Rows[row] != nil && len(sh.Rows[row].Cells) > col {
-		return sh.Rows[row].Cells[col]
+	// If the user requests a row beyond what we have, then extend.
+	for len(sh.Rows) <= row {
```

Comment:
This code was put in place because when you merge cells, ghost cells have to be generated underneath the area covered by the merge.

As far as I understand, the library is not set up to deal with a sparse matrix, so there is no mechanism to create a cell at, say, 100,000x100,000 and not backfill all the other cells, since cells are addressed via array indexes.

So yes, a consequence of this is that if someone puts a cell down at 100,000 you're going to have a big sheet generated. The way it was before, it would just crash because it would try to address way beyond the end of the slice.

Unless, of course, I am missing something regarding the core of how rows and cells are accessed.